### PR TITLE
disable export button for cases

### PR DESCRIPTION
**Pull request by ghazwarhili**

### CODE REVIEW

**Comment by basseche** [comment]
@ghazwarhili you have conflicts on this PR can you please resolve them

**Review by basseche — APPROVED** [review]
it's ok. You just have to resolve conflicts

File: src/components/menus/content-contextual-menu.tsx [hunk 1]
```diff
@@ -605,8 +605,8 @@ const ContentContextualMenu = (props: ContentContextualMenuProps) => {
                 icon: <FileDownload fontSize="small" />,
             });
         }
-
-        if (allowsExportCase()) {
+        // disabled until fix bug from the back side
+        if (!allowsExportCase()) {
```
Comment (etiennehomer) [inline src/components/menus/content-contextual-menu.tsx]: The condition is not good. Keep the same condition and comment all the code nope ?

**Review by etiennehomer — CHANGES_REQUESTED** [review]

File: src/components/menus/content-contextual-menu.tsx (hunk 1 above)
Comment (ghazwarhili) [reply src/components/menus/content-contextual-menu.tsx]: okey

**Review by etiennehomer — APPROVED** [review]

**Pull request merged**